docs/Queryable-Extensions.md

```diff
@@ -223,4 +223,4 @@ Not supported:
 * Value converters
 * **Any calculated property on your domain object**
 
-Additionally, recursive or self-referencing destination types are not supported as LINQ providers do not support this. Typically hierarchical relational data models require common table expressions (CTEs) to correctly resolve a recursive join.
+Additionally, recursive or self-referencing destination types are not supported as LINQ providers do not support this. Typically hierarchical relational data models require common table expressions (CTEs) to correctly resolve a recursive join. Finally, EntityFramework (Core) inheritance is also not supported, as it is not supported by LINQ.

```

Comment:
I think we can remove the existing paragraph. That's mostly true for the old EF, but not for EF Core, and we mainly target EF Core these days.

Reply:
So how do you suggest this paragraph should read?

Reply:
https://github.com/AutoMapper/AutoMapper/pull/4097/commits/b5712e23de2089d0ce2de70608c496d32a88febb
I've already changed that file. But thanks!